docusaurus.config.js

```diff
@@ -52,46 +52,82 @@ export default {
 			{
 				specs: [
 					{
-						id: 'WebserviceStandard',
-						spec: 'https://demo.yetiforce.com/api/WebserviceStandard.json',
-						url: 'https://demo.yetiforce.com/api/WebserviceStandard.json',
+						id: 'WebserviceStandard_7x',
```

Comment:
Nie może być nazwa 7x musi być konkretną wersja bo jak będziemy scalać nowe api do promesy to tam jest sporo zmian